base/src/commonTest/kotlin/IsOnMainThreadTest.kt

```diff
@@ -0,0 +1,35 @@
+import com.splendo.kaluga.base.isOnMainThread
+import com.splendo.kaluga.base.runBlocking
+import kotlinx.coroutines.Dispatchers
+import kotlin.test.Test
+import kotlin.test.assertFalse
+import kotlin.test.assertTrue
+
+/*
+ * Copyright 2022 Splendo Consulting B.V. The Netherlands
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ */
+
+class IsOnMainThreadTest {
```

Comment:
Similar test can be made for android (simulator only)

Reply:
And I suppose JVM.

Reply:
Its already in commonMain so should be running for all platforms

Reply:
I misread this as iOS Main Thread Test 🤦